bindings/rust/s2n-tls/src/testing.rs

```diff
@@ -77,7 +77,7 @@ pub trait Context {
 
 pub enum Mode {
     Client,
-    Server,
+    _Server,
```

Comment:
From offline discussion: let's actually remove these if they're no longer used. It doesn't seem off topic to me, and the underscore doesn't make a better diff than a removal would :)